discovery/gossiper.go

```diff
@@ -1292,6 +1292,26 @@ func (d *AuthenticatedGossiper) processChanPolicyUpdate(
 			return nil, err
 		}
 
+		// We'll avoid broadcasting any updates for private channels to
+		// avoid directly giving away their existence. Instead, we'll
+		// send the update directly to the remote party.
+		if edgeInfo.info.AuthProof == nil {
+			remotePubKey := remotePubFromChanInfo(
+				edgeInfo.info, chanUpdate.ChannelFlags,
+			)
+			err := d.reliableSender.sendMessage(
+				chanUpdate, remotePubKey,
+			)
+			if err != nil {
+				log.Errorf("unable to reliably send %v for "+
```

Comment:
nit: log message should be capitalized

Reply:
What log message? We never capitalize log messages typically.

Reply:
we've been applying this heuristic for the past 8 months at least